core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -311,6 +312,29 @@ public void format() throws IOException {
         .toString());
   }
 
+  /**
+   * Triggers a checkpoint in this ufs journal.
```

Comment:
"Triggers" suggests asynchronous, I think "Creates" is a better word here

Reply:
Changed the `triggers` to `creates` to sound more synchronous